selfservice/flow/settings/handler.go

```diff
@@ -406,7 +415,13 @@ func (h *Handler) fetchFlow(w http.ResponseWriter, r *http.Request) error {
 		return errors.WithStack(herodot.ErrForbidden.WithID(text.ErrIDInitiatedBySomeoneElse).WithReasonf("The request was made for another identity and has been blocked for security reasons."))
 	}
 
-	if err := h.d.SessionManager().DoesSessionSatisfy(r, sess, h.d.Config().SelfServiceSettingsRequiredAAL(r.Context())); err != nil {
+	// we cannot redirect back to the settings flow
```

Comment:
I'm not sure if I understand what the comment is saying. Could you clarify? 

If I understand:

1. generate url `/ui/settings?flow=...`
2. make DoesSessionSatisfy include it in the return URL `/ui/login?return_to=/ui/settings?flow=...`

Reply:
Yes exactly, since the getSettingsFlow always returns JSON, you cannot redirect to it. 

I will adjust the comments to make it more apparent